specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -0,0 +1,686 @@
+            "$ref": "./examples/ForwardGeocodingBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "geocodingBatchRequestBody",
+            "in": "body",
+            "description": "The list of address geocoding queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "206": {
+            "description": "Partial Content. Api would return 206 when part of the batchItem fail to get a response.",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "GeoJSONPoint": {
+      "type": "object",
+      "description": "A structure for Point Geometry",
+      "properties": {
+        "type": {
+          "description": "Specifies the `type` for the geometry. Value should always be equal to \"Point\".",
+          "type": "string",
+          "enum": [
+            "Point"
+          ]
+        },
+        "coordinates": {
+          "description": "Coordinates for the `Point` geometry.",
+          "type": "array",
+          "minItems": 2,
+          "maxItems": 2,
+          "items": {
+            "type": "number",
+            "format": "double"
+          }
+        }
+      }
+    },
+    "Bbox": {
+      "description": "information on the coordinate range for point Geometry.  Please refer to [RFC 7946](https://datatracker.ietf.org/doc/html/rfc7946#section-5) for details.",
+      "type": "array",
+      "minItems": 4,
+      "maxItems": 4,
+      "items": {
+        "type": "number",
+        "format": "double"
+      }
+    },
+    "UsageType": {
+      "type": "string",
+      "enum": [
+        "Display",
+        "Route"
+      ]
+    },
+    "GeocodePoints": {
+      "description": "A collection of geocode points that differ in how they were calculated and their suggested use. For a description of the points in this collection, see the Geocode Point Fields",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "geometry": {
+            "$ref": "#/definitions/GeoJSONPoint"
+          },
+          "calculationMethod": {
```

Comment:
We can give more details as in BME: https://docs.microsoft.com/en-us/bingmaps/rest-services/locations/location-data (search for calculationMethod)

Reply:
updated

Reply:
There is a reference to a non-existing section in the document: "[...] see the Geocode Point Fields".
Please change to "[...] see below".

Reply:
After checking the preview page, I think "For a description of the points in this collection, see the Geocode Point Fields" is redundant.
![image](https://user-images.githubusercontent.com/86343602/144168036-79f1b5d0-4517-490d-9092-6487779b3f9d.png)
in the section above, client can easily see the sub fields' description.
![image](https://user-images.githubusercontent.com/86343602/144168119-1a627f65-2a7e-4e19-97a4-1217f838d10a.png)
in this one, client can check the detail by clicking the url.